### PR TITLE
Ontology implementation

### PR DESCRIPTION
**Pull request by lwasser**
hey @mbjoseph please note this PR just has the code fixes. Have a look and see what you think.
because i changed some DIR names and file names git sees them as new files. I actually don't know how to get around that in Git but if you do please let me know.

You can't test this code until i push another HUGE set of minor edits to all of my course lessons. i just thought 40 pages was a big enough start for you to review.

If you are ok with these edits, please let me know and i'll push all of the lesson changes so you can see it in action locally. 

Don't worry i'm working late because i did a bunch of running the past few mornings and i code best at night... hence my crazy schedule.

### CODE REVIEW

**Comment by lwasser** [comment]
note i've added 2 more commits that reflect the final cleaned dir. this reflects changes to all of the posts but as individual commits so you can ignore them and simply see how the branch builds

File: _posts/course-materials/earth-analytics/week-12/in-class/2017-04-19-social-media-03-r.md [hunk 1]
```diff
@@ -118,7 +114,9 @@ those.
 
 # remove http elements manually
 climate_tweets$stripped_text <- gsub("http.*","",  climate_tweets$text)
+## Error in gsub("http.*", "", climate_tweets$text): object 'climate_tweets' not found
```
Comment (mbjoseph) [inline _posts/course-materials/earth-analytics/week-12/in-class/2017-04-19-social-media-03-r.md]: Is this error and the ones below going to be fixed with the other changes you mentioned?

File: _posts/course-materials/earth-analytics/week-8/in-class/2017-03-08-automation02-function-example-r.md [hunk 2]
```diff
@@ -44,7 +46,28 @@ setwd("~/Documents/earth-analytics")
 
 # load spatial packages
 library(raster)
+## Loading required package: sp
```
Comment (mbjoseph) [inline _posts/course-materials/earth-analytics/week-8/in-class/2017-03-08-automation02-function-example-r.md]: You ok with this stuff being in the post too?

**Comment by lwasser** [comment]
hey @mbjoseph -- great! i'm working on lesson development separately and those two sets are on my radar but NO i didn't see that error in the lessons so THANK YOU for pointing it out :) i'll clean them up today as i do another swee

File: _posts/course-materials/earth-analytics/week-8/in-class/2017-03-08-automation02-function-example-r.md (hunk 2 above)
Comment (lwasser) [inline _posts/course-materials/earth-analytics/week-8/in-class/2017-03-08-automation02-function-example-r.md]: @mbjoseph are you talking about it not loading "quietly"? i started to add a general "hide messages" statement at the top of each lesson - but my guess is i didn't get to these lessons yet as I know i need a gentle redo of them to add the "overlay()" raster processing function!! i'll add these things to my todo's in the issues

File: _posts/course-materials/earth-analytics/week-12/in-class/2017-04-19-social-media-03-r.md (hunk 1 above)
Comment (lwasser) [inline _posts/course-materials/earth-analytics/week-12/in-class/2017-04-19-social-media-03-r.md]: weird -- 

https://earthlab.github.io/course-materials/earth-analytics/week-12/text-mining-twitter-data-intro-r/

the live lesson looks ok.

Comment (lwasser) [inline _posts/course-materials/earth-analytics/week-12/in-class/2017-04-19-social-media-03-r.md]: lemme see if things are synced properly

File: _posts/course-materials/earth-analytics/week-8/in-class/2017-03-08-automation02-function-example-r.md (hunk 2 above)
Comment (lwasser) [inline _posts/course-materials/earth-analytics/week-8/in-class/2017-03-08-automation02-function-example-r.md]: Ok cool it builds now after feeding R some cookies.
the word network is way uglier so i may fix that but again i'll wait till i go back to the lessons again!! they are slated for an update. They key for this PR was really the code changes to the side bar to reflect the new ontology @mbjoseph ! i'm still redoing my class :)  but these are good catches!

**Pull request merged**